### ISSUE TITLE
Can't find hawk-X.Y.Z-beta.tar

### ISSUE THREAD

**Issue by sph17** [atifrahman/HAWK#19]
Hello,

I'm trying to install HAWK 1.5.0. I downloaded with git clone but cannot find any file that corresponds to hawk-X.Y.Z-beta.tar?? 
I tried unzipping the file and make from the master folder, but none of the run files seem to be in the folder either. Could you please give guidance?

**Comment by atifrahman** [comment]
The hawk-1.5.0-beta.tar.gz file is available here
https://github.com/atifrahman/HAWK/releases

**Comment by sph17** [comment]
Thanks for the link. I followed the installation but can only find `runHawk_old` file that is executable. Same think with the other files such as `countKmers_jf1` and `countKmers_jf2`. Wondering if I am not installing this correctly..

**Comment by atifrahman** [comment]
Did you run make? Please see the instructions here
https://github.com/atifrahman/HAWK/blob/master/README.md

**Comment by sph17** [comment]
Yes, I ran make and followed the instructions but did not make any of the run files mentioned after installation. The only run files I find are in the supplement.

<img width="692" alt="Screen Shot 2020-04-27 at 11 08 20 AM" src="https://user-images.githubusercontent.com/64153168/80388069-709c5200-8877-11ea-97b9-692fbcbf16d4.png">
